db/system_keyspace.cc

```diff
@@ -2642,6 +2655,304 @@ class clients_table : public streaming_virtual_table {
     }
 };
 
+class cluster_description_table : public streaming_virtual_table {
```

Comment:
Fixed the commit message and described those virtual tables in mentioned doc.

Reply:
Those tables don't have the structure and schema that the DESC commands expect. Their content has to be aggregated and massaged to get to the desired output. This is all the newly added virtual tables do.

Reply:
I know. I'm just saying that proposing the new virtual table itself (as opposed to the DESC command) as an API is weird, because we already have a different table with the same information, so why would the user need it? I understand why DESC wants it, but not why an end-user will want to use this virtual table.